docs/products/postgresql/concepts/pg-audit-logging.md

```diff
@@ -0,0 +1,196 @@
+[Aiven API](https://api.aiven.io/doc/), the [Aiven CLI](/docs/tools/cli), or
+[psql](https://www.postgresql.org/docs/current/app-psql.html).
+
+### Configuration options
+
+When enabled on your service, the audit logging can be
+[configured](/docs/products/postgresql/howto/use-pg-audit-logging) to match your use case.
+[Audit logging parameters](https://github.com/pgaudit/pgaudit/tree/6afeae52d8e4569235bf6088e983d95ec26f13b7#readme)
+for fine-tuning the feature are the following:
+
+- `pgaudit.log` (default: none)
+  Classes of statements to be logged by the session audit logging
+
+- `pgaudit.log_catalog` (default: on)
+  Whether the session audit logging should be enabled for a statement with all relations
+  in `pg_catalog`
+
+- `pgaudit.log_client`
+  Whether log messages should be visible to a client process, such as `psql`
+
+- `pgaudit.log_level`
+  Log level that should be used for log entries
+
+- `pgaudit.log_parameter` (default: off)
+  Whether audit logs should include the parameters passed with the statement
+
+- `pgaudit.log_parameter_max_size`
+  Maximum size (in bytes) of a parameter's value that can be logged
+
+- `pgaudit.log_relation` (default: off)
+  Whether a separate log entry for each relation (for example, TABLE or VIEW) referenced
+  in a SELECT or DML statement should be created
+
+- `pgaudit.log_rows`
+  Whether the audit logging should include the rows retrieved or affected by a statement
+  (with the rows field located after the parameter field)
+
+- `pgaudit.log_statement` (default: on)
+  Whether the audit logging should include the statement text and parameters
+
+- `pgaudit.log_statement_once` (default: off)
+  Whether the audit logging should include the statement text and parameters in the first
+  log entry for a statement/ sub-statement combination (as opposed to including them in
+  all the entries)
+
+- `pgaudit.role`
+  Master role to use for an object audit logging
+
+:::note[Full list of audit logging parameters]
+For information on all the configuration parameters, preview
+[Settings](https://github.com/pgaudit/pgaudit/tree/6afeae52d8e4569235bf6088e983d95ec26f13b7#readme).
+:::
+
+### Collecting and visualizing logs
+
+You can
+[access your collected audit logs](/docs/products/postgresql/howto/use-pg-audit-logging)
+either directly in the log output of your Aiven for PostgreSQL service or by integrating
+with another service that allows monitoring and analyzing logs, such as Aiven for
+OpenSearch®. To
+[visualize your audit logs](/docs/products/postgresql/howto/use-pg-audit-logging), you can
+use [OpenSearch Dashboards](/docs/products/opensearch/dashboards).
+
+### Disabling audit logging
+
+To
+[disable the audit logging on your service (database)](/docs/products/postgresql/howto/use-pg-audit-logging),
+you need to modify your service's advanced configuration with the
+[Aiven Console](https://console.aiven.io), the [Aiven API](https://api.aiven.io/doc/), the
```

Comment:
**comment** we still haven't enabled pgaudit tunning through Aiven API,  so would not include this for now